config/deploy/production.rb

```diff
@@ -0,0 +1,6 @@
+user = 'raymoond'
+hostname = 'coursemology.org'
+
+server hostname, user: user, roles: %w{web app db}, primary: true
+
+set :rails_env, 'production'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.